android/src/main/java/com/reactbolt/sdk/RNBoltReactLibraryModule.java

```diff
@@ -256,6 +257,8 @@ public void discoverDevice() {
         }
         CCConsumerApi api = CCConsumer.getInstance().getApi();
 
+        debug("starting discovery");
```

Comment:
Is it cool to keep all these debug messages?

Reply:
I think so, but am open to other opinions.  You can [toggle debugging on and off](https://github.com/BigRoomStudios/react-native-card-connect/blob/4c493761ac1768a9258f8ed49d41eb1bd211fef0/android/src/main/java/com/reactbolt/sdk/RNBoltReactLibraryModule.java#L71) from JS, and the debug function will only [log something if debugging is enabled](https://github.com/BigRoomStudios/react-native-card-connect/blob/4c493761ac1768a9258f8ed49d41eb1bd211fef0/android/src/main/java/com/reactbolt/sdk/RNBoltReactLibraryModule.java#L319).  I found that as soon as I thought I was done with a debug message and removed it, I'd just want it again later and would add it back in.  This makes it easy to toggle them all on and off (which the card connect native SDK's also support).

Reply:
Sweet, let's leave 'em if they're useful for future debugging.